parser.py

```diff
@@ -59,6 +61,26 @@ def import_module(module):
         return importlib.import_module(module)
 
 
+import_module('dataclasses') # must be installed for python 3.6, is bundled since 3.7
```

Comment:
Please extend the comment in the readme about modules to be installed.

Please add to the comment in the import section at the top.